securesystemslib/signer/_sigstore_signer.py

```diff
@@ -159,16 +158,32 @@ def from_priv_key_uri(
             raise ValueError(f"SigstoreSigner does not support {priv_key_uri}")
 
         params = dict(parse.parse_qsl(uri.query))
+        ambient = params.get("ambient", "true") == "true"
 
-        if params.get("ambient") == "false":
+        if not ambient:
```

Comment:
This the change of behaviour intended? Before, only "false" disabled "ambient", now any value but "true" will disable.

Both seems fine, it should just be clear to the user, and preferably align with expected behaviour.

Reply:
oh yeah I should have commented on that. It is intended (it felt more logical and the code looked better) but I don't feel at all strongly about it